src/msbuild/Directory.Build.props

```diff
@@ -41,6 +41,7 @@
     <PackageTags>MSBuild</PackageTags>
 
     <TargetMSBuildToolsVersion>Current</TargetMSBuildToolsVersion>
+    <PublishWindowsPdb Condition="'$(DotNetBuildOrchestrator)' == 'true'">false</PublishWindowsPdb>
```

Comment:
Can you add a comment to these indicating why the Orchestrator context is important?